package/piwebcam/piwebcam.mk

```diff
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-PIWEBCAM_VERSION = 29f6923730801980b48de444a362519728a62d7c
-PIWEBCAM_SITE = git://github.com/showmewebcam/uvc-gadget.git
+PIWEBCAM_VERSION = 4a025fc8911b0cb2bd49e6a3ca921b56006317f0
+PIWEBCAM_SITE = git://github.com/peterbay/uvc-gadget.git
```

Comment:
I wonder if we should cherry-pick @peterbay's commits and port it over here so we can have better control of what's going on in the long run.

Reply:
It might make sense to combine these two repositories into one?

Reply:
Yeah let's direct it to @peterbay's repo.